app/models/forward_call.rb

```diff
@@ -0,0 +1,14 @@
+class ForwardCall < ApplicationRecord
+    
+    def initialize(number)
+        @number = number
+    end 
+    
+    def message
+        'Forwarding your call now.'
+    end 
+    
+    def number
```

Comment:
You can replace this by adding `attr_reader :number` to the class. Usually these are declared at the top, somewhere between the `class` and `def initialize` lines. [Checkout this article for details about attr_* methods.](https://mixandgo.com/learn/ruby_attr_accessor_attr_reader_attr_writer)